compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/irCodegenUtils.kt

```diff
@@ -88,7 +89,10 @@ fun JvmBackendContext.getSourceMapper(declaration: IrClass): SourceMapper {
     // NOTE: apparently inliner requires the source range to cover the
     //       whole file the class is declared in rather than the class only.
     // TODO: revise
-    val endLineNumber = fileEntry?.getSourceRangeInfo(0, fileEntry.maxOffset)?.endLineNumber ?: 0
+    val endLineNumber = when {
+        fileEntry is MultifileFacadeFileEntry -> 0
```

Comment:
I think it should be `-1`. In the next few lines this value is being evaluated into line count so having 0 lines in such case seems more reasonable for me.

Reply:
I chose zero there because that's what I believe the old code was doing, and I wanted to preserve the original behavior.  I believe previously fileEntry was null, the null propagated through the original expression and fell into the `?: 0` at the end of the line.  With my change, fileEntry is no longer null, so we just needed to avoid calling `getSourceRangeInfo` but still return zero.

I'm happy to change it to `-1` if you'd prefer, but just note that I think that would be a behavioral change.  Would you like me to update the PR?

Reply:
Also if you would prefer I change it to `-1`, I assume I should change it for both cases (including the `?: 0`)?

Reply:
cc @romanart

Reply:
Yeah, I see. Thank you for explanation. Will run CI and merge it.